playwright-e2e/page-objects/local-authority/laBirthCertificateDetailsSection.page.ts

```diff
@@ -0,0 +1,107 @@
+import { Locator, Page } from '@playwright/test';
+
+export class LAChildSexAtBirthPage {
+  readonly childSexAtBirthHeading: Locator;
+
+  readonly maleRadioButton: Locator;
+  readonly femaleRadioButton: Locator;
+  readonly otherRadioButton: Locator;
+
+  readonly otherSexLabel: Locator;
+
+  readonly saveAndContinueButton: Locator;
+  readonly saveAsDraftButton: Locator;
+
+  constructor(page: Page) {
+    this.childSexAtBirthHeading = page.getByRole('heading', { name: "What was the child's sex at" });
+
+    this.maleRadioButton = page.getByLabel('Male', { exact: true });
+    this.femaleRadioButton = page.getByLabel('Female', { exact: true });
+    this.otherRadioButton = page.getByLabel('Other', { exact: true });
+
+    this.otherSexLabel = page.getByLabel("For example, if the child's");
+
+    this.saveAndContinueButton = page.getByRole('button', { name: 'Save and continue' });
```

Comment:
these buttons are already defined here https://github.com/hmcts/adoption-web/blob/dea8ca488600ab3b27e37314af2385fa3ec51968/playwright-e2e/pages/basePage.page.ts#L21

You can re-use them